server/routes/api/place.py

```diff
@@ -124,6 +125,59 @@ def child(dcid):
     return json.dumps(child_places)
 
 
+# Defines the map from higher geos to their respective subgeos.
+SUB_GEO_LEVEL_MAP = {
+    "Country": "AdministrativeArea1",
+    "AdministrativeArea1": "AdministrativeArea2",
+    "AdministrativeArea2": "City"
+}
+@bp.route('child/statvars/<path:dcid>')
```

Comment:
Please move this to choropleth.py so that we centralize all the choropleth related API calls.

Reply:
Also, statvars should be singular, since only one is accepted (we've also been mostly using statsvar, rather than statvar)

Reply:
This is not choropleth specific though necessarily. It just returns all the statistical variables that are valid for subgeos of a particular place which seems to fit much of the functionality in this file. I'm not sure what you mean by the second comment. This function returns many statsvar not just one.

Reply:
Oh, yes, my bad. It does return all stat vars available.

The functionality is choropleth specific in that it the level containment, geo levels, etc are specific to what choropleth needs. Other functionality here are place page specific. I'd like to keep choropleth changes contained to a few files to ease maintainability.